core/domain/story_services_test.py

```diff
@@ -365,6 +365,22 @@ def test_deassigning_manager_role(self):
         self.assertFalse(story_services.check_can_edit_story(
             self.user_b, story_rights))
 
+    def test_invalid_dict_and_no_changes(self):
```

Comment:
I'm still not quite sure that these tests fully cover the various possibiliities. I noticed another omission in assign_role().

Could you please confirm that the coverage for assign_role() in story_services_test is 100%? Instructions for generating a coverage report are [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests).

Reply:
Yes, @seanlip I checked the test coverage for assign_role() and it's 100%.`assign_role()` is from line 812-869
![screenshot from 2018-11-21 20-20-35](https://user-images.githubusercontent.com/20617436/48848877-1f0eae80-edcb-11e8-9e55-e7d9ad6f2a41.png)